common/app/views/emailEmbed.scala.html

```diff
@@ -0,0 +1,30 @@
+@(metaData: model.Page)(body: Html)(implicit request: RequestHeader)
+
+@import conf.switches.Switches._
+@import common.InlineJs
+@import templates.inlineJS.blocking.js._
+
+<!doctype html>
+<head>
+    @if(play.Play.isDev() || !InlineCriticalCss.isSwitchedOn) {
```

Comment:
I guess you wouldn't want this switch to prevent putting the css int the head, because your inlining would then break?